stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/PlainTextWithFileAttachmentsViewHolder.kt

```diff
@@ -108,6 +94,41 @@ internal class PlainTextWithFileAttachmentsViewHolder(
         }
     }
 
+    private fun setupAttachments(data: MessageListItem.MessageItem) {
+        binding.attachmentsContainer.removeAllViews()
```

Comment:
A lot of doublings as an intermediate step for the solution. In next PR I will fuse these VHs into one